images/spire/configs/latest.agent.apko.yaml

```diff
@@ -18,17 +12,13 @@ accounts:
     - username: nonroot
       uid: 65532
       gid: 65532
-
-# The spire-agent must run as root currently.
-# See https://github.com/spiffe/spire/issues/1862 for context
+  # The spire-agent must run as root currently.
+  # See https://github.com/spiffe/spire/issues/1862 for context
+  run-as: root
```

Comment:
this comment was here prior, but the helm chart failed to deploy until I added the `run-as: root` 🤔

Reply:
Sounds to me like the helm test caught a bug! 😁